c7n/resources/account.py

```diff
@@ -166,6 +166,8 @@ class CloudTrailEnabled(Filter):
                     global-events: true
                     multi-region: true
                     running: true
+                    include-management-events: true
+                    log-metric-filter-pattern: "{ ($.eventName = \\"ConsoleLogin\\") }"
```

Comment:
let's add something in the docstring that describes what this is doing: checks for the existence of a cloudwatch alarm and a corresponding SNS subscription for a filter pattern

Reply:
done